### DISCUSSION TITLE
Remote PC won't sleep/suspend

### DISCUSSION THREAD

**Discussion by workdowg** [rustdesk/rustdesk#8473] [Q&A] [answered]
Win11 - When I'm done sometimes I like to put the PC back to sleep (I just got it to start WOL). When I click on the start menu-Power-sleep, all that happens on the remote is the monitor goes off. I see allow restart is in the Rustdesk setting but nothing about sleep/suspend. TIA

**Comment by rustdesk** [comment, marked as answer]
@21pages let's modify our connection wakelock to allow sleep/hibrate manually.

fixed in https://github.com/rustdesk/rustdesk/pull/8504

**Reply by workdowg** [reply]
THX!

**Comment by xlionjuan** [comment]
Mine won't go to sleep too when I click **"Sleep"**, it just nothing happened, I thought it is Windows's problem, not I know it is not
